### DISCUSSION TITLE
When should you not use GitHub GraphQL API?

### DISCUSSION THREAD

**Discussion by rukivbruki** [graphql/graphiql#1830] [General] [answered]
Much has been written about the benefits of the GitHub GraphQL API. And this is a really great technology. The only thing I can't figure out is in what situations is it still better to use the good old REST API v3?

**Comment by rafalp** [comment, marked as answer]
This repository is not associated with GitHub API in any capacity. See [here](https://github.com/graphql/graphiql/discussions/1737#discussioncomment-219863).
